R-package/README.md

```diff
@@ -561,3 +561,22 @@ Use the following link to report bugs/issues,
 
 [https://github.com/RGF-team/rgf/issues](https://github.com/RGF-team/rgf/issues)
 <br>
+
+### **Citation:**
+
+If you use the code of this repository in your paper or research please cite both **RGF** and the **original articles / software** [https://cran.r-project.org/web/packages/RGF/citation.html](https://cran.r-project.org/web/packages/RGF/citation.html):
+
+<br>
+
+```R
+@Manual{,
+  title = {{RGF}: Regularized Greedy Forest},
+  author = {Lampros Mouselimis and Ryosuke Fukatani and Nikita Titov
+    and Tong Zhang and Rie Johnson},
+  year = {2021},
+  note = {R package version 1.0.8},
+  url = {https://CRAN.R-project.org/package=RGF},
+}
+```
```

Comment:
Are you sure we need to include this snippet here in static README file? I'm asking because this adds one more place where we need to bump **year** and **version** manually with every new release. Maybe it is enough to have only the link above to CRAN page with this information where fields are computed automatically with each upload?

Reply:
We don't necessarily need to update this in every version. This code snippet is in the README.md file to make sure the R package is cited in papers or research regardless of the mentioned year or version.
I did also my search on the web on updating these variables automatically for every new version but although it might be feasible it requires additional code and files which are not worth the effort in order to update the year and version every time we update the package on CRAN.

Reply:
OK, I see.
Can we remove this code snippet from README and leave only `CITATION` file with a reference to it from README?